src/components/course/CourseContextProvider.jsx

```diff
@@ -46,6 +50,8 @@ export const CourseContextProvider = ({
     coursePrice,
     currency,
     canOnlyViewHighlightSets,
+    formSubmissionError,
+    setFormSubmissionError,
```

Comment:
[curious] Was eslint complaining about not including `setFormSubmissionError` in this dependency array? Usually state setter functions from `useState` don't need to go in the dependency array as its known to be memoized by React.

Reply:
Yep it was complaining! I removed them from `CourseContextProvider.propTypes` so maybe that will resolve it?